### ISSUE TITLE
Powershell: npm start fails with 'sleep' is not recognized as an internal or external command 

### ISSUE THREAD

**Issue by asharonbaltazar** [dance2die/cra-template-tailwindcss-typescript#13]
First of all, great project. 
Secondly, using bash on Windows successfully open `localhost:3000`. However, the command fails when using Powershell. I'm not sure whether this is a Powershell-specific error or not. 

Here's the whole error:
![image](https://user-images.githubusercontent.com/58940073/102559839-6cd36f00-4096-11eb-91e6-e5b0e3e7e9a3.png)

**Comment by dance2die** [comment]
Thank you @asharonbaltazar for the compliment :)

Yes. `sleep` is a *NIX command. If you are running in `bash` it shoulda work (but not sure if it needs Linux Subsystem for Windows).  
If you are using Windows, you can probably alias [Start-Sleep](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/start-sleep?view=powershell-7.1) PowerShell command to `sleep` as a workaround.

In the mean time I'd have to find a `cross-env` equivalent of `sleep` so it works cross-platform.

**Comment by austincollinpena** [comment]
Why is the sleep even necessary?

**Comment by dance2die** [comment]
> Why is the sleep even necessary?

TL;DR: It's a lazy hack.

There is a brief moment when TW hasn't yet finished converting utilities.  
You can simply refresh w/o `sleep` but I added random sleep time to wait for it.

I am using `run-p` to run `watch:css` and `react-scripts start` in parallel. If I had a used `run-s`, `watch:css` would never finish.

**Comment by akj1608** [comment]
how you fix this error..i am getting the same error

**Comment by dance2die** [comment]
Hi @akj1608. 

Is the error message the same?(plz post the error log if you can) It should have been fixed in https://github.com/dance2die/cra-template-tailwindcss-typescript/issues/19

No access to windows now so can't try it...

**Comment by akj1608** [comment]
0 info it worked if it ends with ok
1 verbose cli [
1 verbose cli   'C:\\Program Files (x86)\\nodejs\\node.exe',
1 verbose cli   'C:\\Program Files (x86)\\nodejs\\node_modules\\npm\\bin\\npm-cli.js',
1 verbose cli   'start'
1 verbose cli ]
2 info using npm@6.14.14
3 info using node@v14.17.5
4 verbose run-script [ 'prestart', 'start', 'poststart' ]
5 info lifecycle mernblog@0.1.0~prestart: mernblog@0.1.0
6 info lifecycle mernblog@0.1.0~start: mernblog@0.1.0
7 verbose lifecycle mernblog@0.1.0~start: unsafe-perm in lifecycle true
8 verbose lifecycle mernblog@0.1.0~start: PATH: C:\Program Files (x86)\nodejs\node_modules\npm\node_modules\npm-lifecycle\node-gyp-bin;C:\Users\lenovo\Desktop\mernblog\node_modules\.bin;C:\Python39\Scripts\;C:\Python39\;C:\Program Files\Common Files\Oracle\Java\javapath;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Windows\System32\OpenSSH\;C:\Program Files\Git\cmd;C:\Program Files\Android\Android Studio\jre\bin;C:\MinGW\bin;C:\Program Files (x86)\nodejs\;C:\ProgramData\chocolatey\bin;D:\Projects\Anaconda;D:\Projects\Anaconda\Library\mingw-w64\bin;D:\Projects\Anaconda\Library\usr\bin;D:\Projects\Anaconda\Library\bin;D:\Projects\Anaconda\Scripts;C:\Program Files\MySQL\MySQL Shell 8.0\bin\;C:\Ruby30-x64\bin;C:\Users\lenovo\AppData\Local\Microsoft\WindowsApps;C:\Users\lenovo\AppData\Local\Programs\Microsoft VS Code\bin;C:\src\flutter\bin;C:\Users\lenovo\AppData\Roaming\npm
9 verbose lifecycle mernblog@0.1.0~start: CWD: C:\Users\lenovo\Desktop\mernblog
10 silly lifecycle mernblog@0.1.0~start: Args: [ '/d /s /c', 'run-p watch:css react-scripts:start' ]
11 silly lifecycle mernblog@0.1.0~start: Returned: code: 1  signal: null
12 info lifecycle mernblog@0.1.0~start: Failed to exec start script
13 verbose stack Error: mernblog@0.1.0 start: `run-p watch:css react-scripts:start`
13 verbose stack Exit status 1
13 verbose stack     at EventEmitter.<anonymous> (C:\Program Files (x86)\nodejs\node_modules\npm\node_modules\npm-lifecycle\index.js:332:16)
13 verbose stack     at EventEmitter.emit (events.js:400:28)
13 verbose stack     at ChildProcess.<anonymous> (C:\Program Files (x86)\nodejs\node_modules\npm\node_modules\npm-lifecycle\lib\spawn.js:55:14)
13 verbose stack     at ChildProcess.emit (events.js:400:28)
13 verbose stack     at maybeClose (internal/child_process.js:1055:16)
13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:288:5)
14 verbose pkgid mernblog@0.1.0
15 verbose cwd C:\Users\lenovo\Desktop\mernblog
16 verbose Windows_NT 10.0.19043
17 verbose argv "C:\\Program Files (x86)\\nodejs\\node.exe" "C:\\Program Files (x86)\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "start"
18 verbose node v14.17.5
19 verbose npm  v6.14.14
20 error code ELIFECYCLE
21 error errno 1
22 error mernblog@0.1.0 start: `run-p watch:css react-scripts:start`
22 error Exit status 1
23 error Failed at the mernblog@0.1.0 start script.
23 error This is probably not a problem with npm. There is likely additional logging output above.
24 verbose exit [ 1, true ]
